cli/src/commands/sidekick.rs

```diff
@@ -201,6 +203,27 @@
 				unreachable!("WaitForLogin should be handled before this")
 			}
 			SubCommand::CheckLoginState => todo!(),
+			SubCommand::GetToken => {
+				struct_fmt::print(
+					&Format::Json,
+					&SideKickResponse::Ok(json!({
+						"token": ctx.access_token,
+					})),
+				)?;
+			}
+		}
+
+		Ok(())
+	}
+
+	pub fn validate_token(&self, token: &Option<String>) -> Result<()> {
+		if token.is_none() {
+			struct_fmt::print(
+				&Format::Json,
+				&SideKickResponse::Err(json!({
+					"error": "No Rivet token found, please do the sign in process",
+				})),
+			)?;
```

Comment:
This is going to get repetitive, maybe add a helper function?

Reply:
This flow is a little weird. We're erroring, but it returns on the OK path. Wondering if instead we should try and downcast the anyhow error in to a `SideKickError` or something, and if not return an internal error. We do something similar with `GlobalError`: try to downcast and return internal if we can't and log it.

Reply:
Good point. I wasn't super happy with how this was going to operate, but this seems like a good path now that we can see how much it will take right now. I'll change this out in the parent PR and trickle it down to the rest.